src/hooks/useApeSnackbar.ts

```diff
@@ -29,7 +29,7 @@ const displayError = (error: any) => {
 export const useApeSnackbar = () => {
   const { enqueueSnackbar } = useSnackbar();
   const showInfo = (message: string) =>
-    enqueueSnackbar(message, { variant: 'default' });
+    enqueueSnackbar(message, { variant: 'info' });
```

Comment:
what does this do?

Reply:
Weirdly, I couldn't figure out how to style the default variant. So I changed to the info variant just to be able to style it